parsigs/parse_sig_api.py

```diff
@@ -86,27 +87,39 @@ def _parse_sig(sig: str, model: Language):
 
     return _create_structured_sigs(model_output)
 
+def _autocorrect(sig):
+    spell = SpellChecker()
```

Comment:
spell checker should be defined once globally

Reply:
Done.